code/setup.dm

```diff
@@ -276,6 +286,27 @@
 #define slot_w_uniform_str	"w_uniform"
 #define icon_head		"slot_head"
 
+
+//itemstate suffixes. Used for containedsprite worn items
+#define WORN_LHAND	"_lh"
```

Comment:
Curious, is this used anywhere else, other than `update_icons.dm`? If not, then please move the #defines there and #undef them at the bottom of the file. Memory management is a good practice.